core/arch/arm/plat-imx/platform_config.h

```diff
@@ -61,22 +61,22 @@
 #define CFG_DDR_START			DDR_PHYS_START
 #define CFG_DDR_SIZE			DDR_SIZE
 
-#ifndef CFG_DDR_TEETZ_RESERVED_START
-#define CFG_DDR_TEETZ_RESERVED_START	0x9E000000
+#ifndef CFG_TZDRAM_BASE
+#define CFG_TZDRAM_BASE			0x9E000000
```

Comment:
Here i would prefer 
```
#if !defined(CFG_TZDRAM_BASE) && defined(CFG_DDR_TEETZ_RESERVED_START)
#define CFG_TZDRAM_BASE			CFG_DDR_TEETZ_RESERVED_START
#endif
#ifndef CFG_TZDRAM_BASE
#define CFG_TZDRAM_BASE			0x9E000000
#endif
```

Reply:
good point.

Reply:
OK, I restored `CFG_DDR_TEETZ_RESERVED_START` and moved the CFG options to conf.mk.